### PR TITLE
docs: update sms and email 2fa docs

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/mfa/30_sms.mdx [hunk 1]
```diff
@@ -68,15 +64,79 @@ To enable MFA via SMS, you need to configure an SMS channel in the Ory configura
 </Tabs>
 ```
 
+4. Ensure your Identity Schema correctly marks one of the traits as the identifier for this method.
```
Comment (jonas-jonas) [inline docs/kratos/mfa/30_sms.mdx:67]:
```suggestion
4. Ensure your Identity Schema correctly marks one of the traits as the identifier for `code` method.
```

File: docs/kratos/mfa/30_sms.mdx [hunk 2]
```diff
@@ -68,15 +64,79 @@ To enable MFA via SMS, you need to configure an SMS channel in the Ory configura
...
-also need to specify which trait to use for delivering the code to the user. Make sure, this trait exists in the identity schema
-and set the `via` parameter to its identifier. For example, if you have a trait called `phone_number`, you'd set `via` to
-`phone_number`.
+To start a new MFA flow, for an already existing session, create a new login flow with the `aal` parameter set to `aal2`:
 
-Ory will return an error in the UI, if the trait does not exist in the identity's schema or the trait is empty in the current
-identity. So make sure this trait is required in your identity schema.
+```
+https://<your-custom-domain-or-project-slug>/self-service/login/browser?aal=aal2
+```
```
Comment (jonas-jonas) [inline docs/kratos/mfa/30_sms.mdx:142]: Should we list SDK examples?

**Review by jonas-jonas — COMMENTED** [review]
LGTM

File: docs/kratos/mfa/30_sms.mdx (hunk 2 above)
Comment (aeneasr) [reply docs/kratos/mfa/30_sms.mdx:142]: Would be nice!

Comment (aeneasr) [reply docs/kratos/mfa/30_sms.mdx:142]: But not possible before I leave:/

**Pull request merged**